linked-list.md

```diff
@@ -1,29 +1,35 @@
-Deques have four fundamental operations (using Array terminology):
+Like an array, a linked list is a simple linear data structure. Several 
+common data types can be implemented using linked lists, like queues, 
+stacks and associative arrays.
 
-* push (insert value at back)
-* unshift (insert value at front)
-* pop (remove value at back)
-* shift (remove value at front)
+A linked list is a collection of data elements called *nodes*. In a 
+*singly linked list* each node holds a value and a link to the next node. 
+In a *doubly linked list* each node also holds a link to the previous 
+node.
 
-The difference between deque and Array performance is that a deque
-implements all of these operations in constant time. Even clever
-implementations of Array will often have to copy the entire Array in
-order to unshift.
+You will write an implementation of a doubly linked list. You will use a
+`Node`-class (or similar) to hold a value and pointers to the next and 
```

Comment:
Can we avoid calling it a "class" by saying something like _you will implement a Node to hold a value and pointers..._?